Library/Formula/ktoblzcheck.rb

```diff
@@ -16,6 +14,16 @@ def install
                           "--prefix=#{prefix}"
     system "make"
     ENV.j1
-    system "make install"
+    system "make", "install"
+  end
+
+  test do
+    result =<<-EOS.undent
+    Bank: 'Bundesbank' at 'Berlin' (10000000)
+    Account: 123456789
+    Result is: (0) Ok
+
+    EOS
+    assert_equal result, shell_output("#{bin}/ktoblzcheck 10000000 123456789")
```

Comment:
Again, I think matching exact string output here will be potentially error-prone. Partial output and/or exit code checking seems enough.